src/Symfony/Bridge/Twig/TwigEngine.php

```diff
@@ -126,7 +126,7 @@ protected function load($name)
         }
 
         try {
-            return $this->environment->loadTemplate((string) $name);
+            return $this->environment->load((string) $name)->unwrap();
```

Comment:
should we actually unwrap here ? This is an internal method, so we should avoid using it if not necessary. Isn't TemplateWrapper enough for the needs of this class ?

Reply:
It is a protected method inside a non-final class. I was a bit afraid of changing its behavior.